integration/integration_test.go

```diff
@@ -2241,10 +2230,25 @@ func trustedClusters(t *testing.T, suite *integrationTestSuite, test trustedClus
 	require.NoError(t, aux.StopAll())
 }
 
-func checkGetClusters(t *testing.T, tun reversetunnel.Server) []reversetunnel.RemoteSite {
-	clusters, err := tun.GetSites()
-	require.NoError(t, err)
-	return clusters
+func waitForClusters(tun reversetunnel.Server, expected int) func() bool {
+	return func() bool {
+		clusters, err := tun.GetSites()
+		if err != nil {
+			return false
+		}
+
+		// Check the expected number of clusters are connected and they have all
+		// connected with the past 10 seconds.
+		if len(clusters) >= expected {
+			for _, cluster := range clusters {
+				if time.Since(cluster.GetLastConnected()).Seconds() > 10.0 {
+					return false
+				}
+			}
+		}
```

Comment:
I don't quite understand why this is here

Reply:
It's to bring the checks from the removed test into integration tests. Make sure no only did the cluster show up, but it connected within the last 10 seconds.